meson.build

```diff
@@ -0,0 +1,41 @@
+project(
+    'erfa', 'c',
+    version: '2.0.0',
+    meson_version: '>=0.37.0',
```

Comment:
Trying this on my laptop, I get warnings that the code uses features from a newer meson,
```
WARNING: Project targets '>=0.37.0' but uses feature introduced in '0.46.0': pkgconfig.generate optional positional argument.
Build targets in project: 3
WARNING: Project specifies a minimum meson_version '>=0.37.0' but uses features which were added in newer versions:
 * 0.46.0: {'pkgconfig.generate optional positional argument'}
```
Since we are introducing it here, I guess no reason not to pick a newer version? Maybe `>=0.55` to have `meson compile` available? In https://github.com/liberfa/pyerfa/issues/91#issuecomment-1256882425, a suggestion for 0.64 is made, but that is not yet out. My debian/testing has 0.63.2; debian/stable has 0.56.

Reply:
This is because I set it to a low version specifically to check what warnings it generated, then after testing that this built okay I forgot to actually bump the version lol.

But of course since this is a demonstration and still needs to be hooked up to CI and get buy-in from stakeholders there's time for that. The most important thing to start with is to get something that works which people can look at.

Reply:
Well, the demonstration is well on its way of convincing me at least! But hooking up to CI would be good.